shared/react-native/ios/Keybase/Storybook.m

```diff
@@ -19,7 +19,7 @@ @implementation Storybook
 - (NSDictionary *)constantsToExport
 {
   // Set this to true to enable storybook mode
-  return @{@"isStorybook": @false};
+  return @{@"isStorybook": @true};
```

Comment:
we don't want to leave this in right

Reply:
ah whoops, the very last thing I tested. thanks!